config.example.toml

```diff
@@ -172,6 +172,8 @@
 
 # rpm_ostree = false
 
+# bootc = false
```

Comment:
It would be great if we can have some doc here:

```suggestion
# For <DISTROS>, if `bootc` is available and this configuration entry is set to true, use
# it to do the update.
# (default: false)
# bootc = false
```